sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md

```diff
@@ -66,3 +66,21 @@ directive:
   transform: >
     $.properties.fields.additionalProperties["x-nullable"] = true;
 ```
+
+``` yaml
+directive:
+  from: swagger-document
+  where: $.definitions.ModelInfo
+  transform: >
+    $.properties.modelId["x-nullable"] = true;
+```
```

Comment:
One extra workaround caused by the swagger and the service behavior being inconsistent. We're treating `CustomFormModelInfo.ModelId` as a string now, instead of a GUID. It's marked as required in the swagger, even though there are scenarios in which no model ID is returned, which makes this workaround necessary for the time being.

Already tracked by https://github.com/Azure/azure-sdk-for-net/issues/13615

Reply:
Could you remind me in which cases we get an empty `ModelId`?

Reply:
Glad you asked! Trying to understand the scenario better, I realized it's an issue with one of our mock tests, and not with the service. I'll get it updated and undo the transform.

Reply:
We were getting a response without a `ModelId`, but in the end it was a mock response created in one of our tests.

Reply:
ohhh nice!!! I like it